### ISSUE TITLE
Flutter's IconButton doesn't match the updated IconButton from Youtube App

### ISSUE THREAD

**Issue by sidrao2006** [flutter/flutter#67038]
Looks like material updated their spec. Atleast for `IconButton`.

Source - YouTube

**Comment by sidrao2006** [comment]
@HansMuller can you please confirm this

**Comment by TahaTesser** [comment]
Hi @sidrao2006 
Can you please provide source? I found this https://material.io/components/buttons#anatomy, not sure what update you talking about tho.
Please look at this issue, https://github.com/flutter/flutter/issues/54776 to learn how new  material spec is compared to current flutter implementation 

Thank you

**Comment by sidrao2006** [comment]
Hi @TahaTesser , yes I do know about this update

**Comment by sidrao2006** [comment]
Some screenshots from Youtube

Youtube follows the material spec. I noticed this change in the last update.

> Note - pictures have been taken from Youtube beta

![20201001_144428.jpg](https://user-images.githubusercontent.com/66567899/94791175-e68f8100-03f4-11eb-8472-ed54918f8adf.jpg)![20201001_144403.jpg](https://user-images.githubusercontent.com/66567899/94791196-eb543500-03f4-11eb-9709-5345698d3725.jpg)![20201001_144310.jpg](https://user-images.githubusercontent.com/66567899/94791211-ef805280-03f4-11eb-98b4-358e9cc7dbe4.jpg)![20201001_144243.jpg](https://user-images.githubusercontent.com/66567899/94791222-f27b4300-03f4-11eb-844c-ad7fcb44ae5c.jpg)

**Comment by sidrao2006** [comment]
[This](https://github.com/flutter/flutter/issues/64702#issuecomment-682625911_) may be a plausible explanation as to why the update hasn't reflected in the official material specs

**Comment by TahaTesser** [comment]
Hi @sidrao2006 
I'm not sure how this issue this different than https://github.com/flutter/flutter/issues/64702? 
Can you please provide details about the screenshot you provided
Otherwise, the IconButton radius thing makes this a  duplicate https://github.com/flutter/flutter/issues/64702
Thank you

**Comment by sidrao2006** [comment]
As of Youtube v15.40.32, the IconButton's splash has updated. #64702 only points out the change in size. Upon further inspection, there is also a change in the ripple effect. As shown in the last image, a light outline is visible post the effect.

However this issue can be transferred to #64702 with confirmation

**Comment by sidrao2006** [comment]
This GIF shows some more instances of this change throughout Youtube

![Youtube Updated Icon Button](https://user-images.githubusercontent.com/66567899/95073005-c24fdf00-0729-11eb-9cbd-ffa6867f597c.gif)

**Comment by sidrao2006** [comment]
@willlarche are there some new changes flutter is missing?

**Comment by willlarche** [comment]
YouTube doesn't follow strict Material Design so if they make choices in their application, it's not an indication that we missed an update on Flutter. In fact, no product follows it 100%. Every product should make the changes they need to make their product work best for them!

On the Ripple animation, @rami-a , was that checked in the latest audit with Design?

**Comment by rami-a** [comment]
Yes I believe the Ripple was checked and it is correct. Like you said Will, this appears to be a case of Youtube making their own choices when it comes to Material Design.

That being said, I believe creating your own custom ink ripple can be achieved by making a subclass of `InteractiveInkFeature` just like `InkSplash` and `InkRipple` and providing it to the splashFactory param of your theme data.
